tests/nlu/test_evaluation.py

```diff
@@ -394,28 +391,26 @@ async def test_eval_data(
         ],
     )
 
-    (_, _, persisted_path) = await train(
-        _config,
-        path=str(tmp_path),
-        data=data_importer,
-        component_builder=component_builder,
-        persist_nlu_training_data=True,
-    )
-
-    interpreter = Interpreter.load(persisted_path, component_builder)
+    with rasa.model.unpack_model(trained_simple_rasa_model) as unpacked_model_directory:
+        _, nlu_model_directory = rasa.model.get_model_subdirectories(
+            unpacked_model_directory
+        )
+        interpreter = Interpreter.load(nlu_model_directory, component_builder)
 
     data = await data_importer.get_nlu_data()
     (intent_results, response_selection_results, entity_results) = get_eval_data(
         interpreter, data
     )
 
     assert len(intent_results) == 46
-    assert len(response_selection_results) == 46
+    assert len(response_selection_results) == 0
     assert len(entity_results) == 46
 
 
 @pytest.mark.timeout(240)  # these can take a longer time than the default timeout
-def test_run_cv_evaluation(pretrained_embeddings_spacy_config: RasaNLUModelConfig):
+def test_run_cv_evaluation(
+    monkeypatch: MonkeyPatch, pretrained_embeddings_spacy_config: RasaNLUModelConfig
```

Comment:
I noticed the spacy model takes quite a while to load etc. How about not using spacy here?